README.md

```diff
@@ -38,21 +40,33 @@ Git Machete IntelliJ Plugin is a port of a great console tool - [git-machete](ht
 Git Machete can help you manage your repository branches and keep it in sync with each other and its counterparts on a remote repository.
 Let's see how this plugin can help you:
 
-### Where to find the plugin tab
+<details>
+<summary>Where to find the plugin tab</summary>
```

Comment:
Pls wrap the summary text into `<b>...</b>` everywhere + `<br/>` after `</summary>`

Reply:
So that the summaries feel more header-like

Reply:
or maybe even `<strong></strong>`